submissions/NwagboJohn/HtmlTask/signUp.html

```diff
@@ -0,0 +1,34 @@
+
+   <!DOCTYPE html>

```

Comment:
please checkout https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/blob/master/STYLEGUIDE.md and use it to format this html file appropriately